test/TestApps/PushNotificationsDemoApp/main.cpp

```diff
@@ -18,7 +18,7 @@ winrt::Windows::Foundation::IAsyncOperation<PushNotificationChannel> RequestChan
     // To obtain an AAD RemoteIdentifier for your app,
     // follow the instructions on https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app
     auto channelOperation = PushNotificationManager::CreateChannelAsync(
-        winrt::guid("ccd2ae3f-764f-4ae3-be45-9804761b28b2"));
+        winrt::guid("0160ee84-0c53-4851-9ff2-d7f5a87ed914"));
```

Comment:
Why the change in the guid? is this intentional or required?

Reply:
Planning to change the GUID as the newer guid works for both scenarios - packaged and unpackaged.